Example/Backpack/UIKit/Components/Badge/BadgesViewController.swift

```diff
@@ -19,24 +19,71 @@
 import UIKit
 import Backpack
 
+private struct BadgeExample {
+    let type: BPKBadgeType
+    let message: String
+    let showOverDarkBackground: Bool
+}
+
 class BadgesViewController: UIViewController {
-    @IBOutlet var badgeCollections: [BPKBadgeContainer]!
-    fileprivate static var badgeTypes = [
-        BPKBadgeType.success, BPKBadgeType.warning,
-        BPKBadgeType.destructive, BPKBadgeType.strong,
-        BPKBadgeType.normal, BPKBadgeType.light,
-        BPKBadgeType.inverse, BPKBadgeType.outline
+    private let badges: [BadgeExample] = [
```

Comment:
simplified the Badge example for UIKit, removing the container class and the Storyboard for it